pkg/featureflag/featureflag.go

```diff
@@ -22,16 +22,35 @@ import (
 	"sync"
 )
 
-var PreviewPrivateDNS = New("PreviewPrivateDNS")
+func Bool(b bool) *bool {
+	return &b
+}
+
+// PreviewPrivateDNS turns on the preview of the private hosted zone support
```

Comment:
All documentation for exported pieces should be readable as English documentation - it makes godoc from the source readable. We're not great at enforcing this or the requirement to document exported pieces, though.